src/webgpu/shader/execution/memory_model/memory_model_setup.ts

```diff
@@ -941,7 +1010,7 @@ const resultShaderCommonCode = [
   shaderEntryPoint,
 ].join('\n');
 
-/**
+/*
```

Comment:
nit: revert this. I believe `/**` is the correct markup for a multiline doxygen-like comment.